Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:16-alpine as runner
+
+WORKDIR /usr/src/pine
+
+# when switching over to node:17 this v may apply
+# ENV NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
Why would it be necessary? I'd rather remove this comment and fix the root cause

Reply:
Fixed in latest node17 version.